src/script/lua_api/l_noise.h

```diff
@@ -150,7 +151,8 @@ class LuaPcgRandom : public ModApiBase
 
 public:
 	LuaPcgRandom(u64 seed) : m_rnd(seed) {}
-	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed, seq) {}
+	// @TODO: set seq
+	LuaPcgRandom(u64 seed, u64 seq) : m_rnd(seed) {}
```

Comment:
do we have a guarantee on the values the `LuaPcgRandom` api returns?
because if so then it can't be replaced